client/src/components/Plastic/PlasticApproach.tsx

```diff
@@ -0,0 +1,65 @@
+import PlasticExampleComp from "./PlasticExampleComp.tsx";
+
+import title from "../../misc/Title";
+import banner from "../../assets/plastic_cleanup_gettyimages-1179252601-1024x683.webp";
+import TopicLink from "./TopicComp.tsx";
+import BannerComp from "./BannerComp.tsx";
+import BackgroundSunComp from "./BackgroundSunComp.tsx";
+
+import innovations from "../../misc/PlasticApproach/Innovations";
+import whatCanWeDoAboutIt from "../../misc/PlasticApproach/WhatCanWeDoAboutIt";
+import cleanUp from "../../misc/PlasticApproach/CleanUp";
+import supportLegislation from "../../misc/PlasticApproach/SupportLegislation";
+import shareKnowledge from "../../misc/PlasticApproach/ShareKnowledge";
+import conclusion from "../../misc/PlasticApproach/Conclusion";
+
+export default function PlasticApproach() {
+  return (
+    <div className="sticky grid grid-cols-11 w-full h-full z-0 overflow-y-auto bg-gradient-c">
+
+      <PlasticExampleComp children={title.divs} />
+
+      <TopicLink
```

Comment:
Nest de Topic links in een 1 component en plaats dat op all je pagina's want nu moet je bij elke wijziging in de topic links veranderen op 3 plaatsen